### PR TITLE
(chore): Add translation support for labels and messages

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary
- This is a follow up on the snackbar PR, it fixes and adds support for multiple language translations

## Screenshots
Screenshot 2024-08-01 at 17 24 47
Screenshot 2024-08-01 at 17 24 38
Screenshot 2024-08-01 at 17 36 23

## Related Issue
-https://openmrs.atlassian.net/browse/O3-3700

### CODE REVIEW

File: translations/en.json [hunk 1]
```diff
@@ -232,23 +262,17 @@
         "stocksource": "",
         "stocksourcedestination": ""
       }
-    },
-    "yes": "Yes"
+    }
   },
   "stockManagement": "Stock Management",
-  "stockManagementLocations": "Locations",
```
Comment (jabahum) [inline translations/en.json:239]: does this mean the sidenav menu items will not be translated ?

Comment (hadijahkyampeire) [reply translations/en.json:239]: Yes, it is now translated

File: translations/ar.json [hunk 2]
```diff
@@ -0,0 +1,292 @@
+{
```
Comment (donaldkibet) [inline translations/ar.json:1]: I might be wrong but we should only provide translation for english then Transifex would provide for the other languages

**Review by donaldkibet — APPROVED** [review]

File: translations/ar.json (hunk 2 above)
Comment (hadijahkyampeire) [reply translations/ar.json:1]: What is Transifex? that would be cool but I remember even on patient-chart, we were manually adding the other language translations but if there is a tool to help in that, let's use it then. @donaldkibet do you know how we can use that Transifex?

Comment (hadijahkyampeire) [reply translations/ar.json:1]: I am gonna go ahead and merge this until we see the Transifex option

**Pull request merged**